cmd/keycloakb/keycloak_bridge.go

```diff
@@ -57,6 +57,7 @@ var (
 
 func init() {
 	rand.Seed(time.Now().UTC().UnixNano())
+	commonhttp.SetEmitter(ComponentName)
```

Comment:
Remove previous declaration of ComponentName, ComponentID, ... and use keycloakb.ComponentName here
(to be checked with @harture to check if these values are used when the CI is building the project)